# HG changeset patch
# User Punarbasu Purkayastha <ppurka@gmail.com>
# Date 1363279594 -28800
# Node ID 97fe6a650f00cf504cf4f7c12200a90b66ceacc8
# Parent  a3b1ac4c81b274e3b1af151ea645ab70a29e3723
doctest to check that the limit is fixed

diff --git a/sage/calculus/calculus.py b/sage/calculus/calculus.py
--- a/sage/calculus/calculus.py
+++ b/sage/calculus/calculus.py
@@ -1165,6 +1165,11 @@
         'below' is deprecated. Please use 'left' or '-' instead.
         See http://trac.sagemath.org/9200 for details.
         -Infinity
+
+    Check that :trac:`12708` is fixed::
+
+        sage: limit(tanh(x),x=0)
+        0
     """
     if not isinstance(ex, Expression):
         ex = SR(ex)
